index.html

```diff
@@ -49,5 +49,165 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
+                                    <button type="button" class="resume-btn">Get My Resume</button>
+                                </div>
+                            </div>
+                            <ul class="about-section-skills">
+                                <li>
+                                <article class="about-card">
+                                        <div class="about-card-image">
+                                           <img src="./img/Icon Langguage.svg" alt="language"> 
+                                        </div>
+                                        <h2>Languages</h2>
+                                        <ul class="about-card-skills">
+                                            <li>javascript</li>
+                                            <li>ruby</li>
+                                            <li>html</li>
+                                            <li>css</li>
+                                        </ul>
+                                    </div>
+                                </article>
+                                </li>
+                            <li>
+                            <article>
+                                <div class="about-card">
+                                    <div class="about-card-image">
+                                       <img src="./img/Icon Name.svg" alt="frameworks"> 
+                                    </div>
+                                    <h2>Frameworks</h2>
+                                    <ul class="about-card-skills">
+                                        <li>bootstrap</li>
+                                        <li>ruby on rails</li>
+                                        <li>RSpec</li>
+                                        <li>Capybara</li>
+                                        <li>Selenium</li>
+                                    </ul>
+                                </div>
+                            </article>
+                        </li>
+                        <li>
+                            <article>
+                                <div class="about-card">
+                                    <div class="about-card-image">
+                                       <img src="./img/Icon Name (1).svg" alt="skills"> 
+                                    </div>
+                                    <h2 id="title-about">Skills</h2>
+                                    <ul class="about-card-skills">
+                                        <li>codekit</li>
+                                        <li>github</li>
+                                        <li>codepen</li>
+                                        <li>gitlab</li>
+                                        <li>terminal</li>
+                                    </ul>
+                                </div>
+                            </article>
+                            </li>
+                            </ul>
```

Comment:
- [ ] As the requirements suggest, you must stick to the design as much as possible (e.g., font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- your design
![image](https://user-images.githubusercontent.com/52143815/192010593-534e5108-23a0-41dc-a514-99ac846ad2a5.png)

- Figma design
![image](https://user-images.githubusercontent.com/52143815/192010657-d8bef478-bfb5-4abf-8884-9e9bb3e993d5.png)